docs/i18n.md

```diff
@@ -0,0 +1,91 @@
+# i18n
+
+## Context
+
+Internationalization on Force was introduced as a hackathon project ([PR](https://github.com/artsy/force/pull/9797)).
+
+We are using the [react-i18next](https://react.i18next.com/) internationalization framework for this.
+
+The goal of these change is separating the text content from the codebase, so copy updates can handled in a smoother process.
+
+See [this
+presentation](https://docs.google.com/presentation/d/1VT44uoGAaHX0EcDaYwGHiY8VU-fAkOXK6rSHto/edit#slide=id.gd1b1ff661f_0_12) about the benefits we expect to achieve.
+
+## How to
+
+Locales are defined on [src/System/locales](https://github.com/artsy/force/tree/main/src/System/locales). Currently, the only language supported is en-US.
+
+### Importing react-i18next
+
+1. Import the library:
+
+```jsx
+import { useTranslation } from "react-i18next”
+```
+
+2. Use the `useTranslation` hook on functional components:
+
+```jsx
+const { t } = useTranslation()
+```
+
+### Adding a static string
+
+1. Add the string to [translation.json](https://github.com/artsy/force/blob/main/src/System/locales/en-US/translation.json) following the Naming conventions.
+2. Use it in any component by calling `{t`<path-for-the-key>`}` (example: `{t`navbar.shows`}`)
```

Comment:
```suggestion
2. Use it in any component by calling ``{t`<path-for-the-key>`}`` (example: ``{t`navbar.shows`}``)
```
I'm pretty sure this is how you write code things that include a backtick `` ` ``